src/Training.AirBnb.Clone.Frontend/src/Modules/profile/components/Darkmode.vue

```diff
@@ -0,0 +1,22 @@
+<template>
+  <div :class="modelValue ? 'theme-bg-primary-toggle' : 'bg-gray-300'" @click="toggle" class="w-10 h-6 flex primary-transition-toggle theme-bg-accent rounded-full p-1 theme-text-primary">
+    <div  :class="modelValue ? 'translate-x-4' : 'translate-x-0'" class="w-4 h-4 primary-transition-toggle bg-textSecondary rounded-full theme-shadow"></div>
```

Comment:
- rename this component to `DarkModeToggler`
- remove toggle value as it uses `AppThemeService` service internally